openid-federation-1_0.xml

```diff
@@ -4499,7 +4499,7 @@ Host: openid.sunet.se
 
       <section title="Trust Mark Status" anchor="status_endpoint">
         <t>
-          This enables an Entity to check whether a Trust Mark has been issued to
+          This enables determining whether a Trust Mark has been issued to
           an Entity and is still active. The query MUST be sent to the Trust Mark Issuer.
```

Comment:
```suggestion
          This enables determining whether a Trust Mark that has been issued to
          an Entity is still active. The query MUST be sent to the Trust Mark Issuer.
```

Reply:
I think we should make it clear (to prevent any confusion that already arised) that this endpoint is to check if a certain trust mark (jwt) is still valid.

Therefore, I suggest rephrasing this part. The previous statement could be read that this endpoint is used to check if the entity has been issued this trustmark. However, since we already passing the trust mark, that part must be true, but we need to check if the trust mark is **still* valid.